clients/itwin/src/Request.ts

```diff
@@ -284,6 +282,8 @@ export async function request(url: string, options: RequestOptions): Promise<Res
   if (options.headers)
     sareq = sareq.set(options.headers);
 
+  sareq = sareq.set(requestIdHeaderName, Guid.createValue());
```

Comment:
TODO: Need to fix this so it only sets the header if it doesn't already have one.

Reply:
fixed.